docs/static/releasenotes.asciidoc

```diff
@@ -87,6 +87,7 @@ This section summarizes the changes in the following releases:
 
 [[notable-8.18.0]]
 ==== Performance improvements and notable issues fixed
+* Fixes an issue introduced in 8.16.4, 8.17.2 where null values were incorrectly accepted for non-nullable settings https://github.com/elastic/logstash/pull/17522[#17522].
```

Comment:
How about removing versions detail?

Reply:
Fixes an issue where null values were incorrectly accepted for mandatory string settings